docs/README.md

```diff
@@ -20,6 +20,11 @@ contributors: false
       </div>
    </div>
 </div>
+
+::: warning Protocol upgrade
+An upcoming protocol upgrade will change the implementation of some of the block properties on zkSync Era. Find more details [in the announcement on GitHub](https://github.com/zkSync-Community-Hub/zkync-developers/discussions/87).
```

Comment:
question: Should we include the dates of expected changes?

Reply:
yeah, it can be useful. On it